src/utils/account-utils.js

```diff
@@ -0,0 +1,22 @@
+import {
+  ACCOUNT_STATUS_ANONYMUS,
+  ACCOUNT_STATUS_JUROR_ACTIVE,
+  ACCOUNT_STATUS_JUROR_INACTIVE,
+} from '../types/account-status-types'
+
+export function getAccountStatus(balances, minActiveBalance) {
+  const { activeBalance } = balances
+
+  if (activeBalance >= minActiveBalance) return ACCOUNT_STATUS_JUROR_ACTIVE
```

Comment:
Shouldn’t we use `BN.js` to compare this?

Reply:
We should